src/sql/workbench/browser/modelComponents/declarativeTable.component.ts

```diff
@@ -316,6 +321,15 @@ export default class DeclarativeTableComponent extends ContainerBase<any, azdata
 		}
 	}
 
+	public onKey(e: KeyboardEvent, row: number) {
+		const event = new StandardKeyboardEvent(e);
+		if (event.equals(KeyCode.Enter) || event.equals(KeyCode.Space)) {
+			this.onRowSelected(row);
+			e.preventDefault();
```

Comment:
FYI there's `EventHelper.stop` that lets you do preventDefault and stopPropagation in one line